components/script/dom/underlyingsourcecontainer.rs

```diff
@@ -218,6 +218,18 @@ impl UnderlyingSourceContainer {
         }
     }
 
+    /// <https://streams.spec.whatwg.org/#dom-underlyingsource-autoallocatechunksize>
+    #[allow(unsafe_code)]
+    pub(crate) fn auto_allocate_chunk_size(&self) -> Option<u64> {
+        match &self.underlying_source_type {
+            UnderlyingSourceType::Js(source, _) => source.autoAllocateChunkSize,
+            UnderlyingSourceType::Tee(_tee_underlyin_source) => {
+                todo!();
```

Comment:
This can be `None` because of https://streams.spec.whatwg.org/#abstract-opdef-createreadablebytestream